### PR TITLE
added contributing.md

### PR DESCRIPTION
**Pull request by ASoTNetworks**

### CODE REVIEW

**Comment by benhylau** [comment]
We already have https://github.com/hyphacoop/remote/blob/master/CONTRIBUTING.md and this is a different file. What are the things you changed, and why not edit existing file?

**Comment by ASoTNetworks** [comment]
I was following the link to /contributing which it 404s so thinking I should create a new file and didn't notice CONTRIBUTING.md

I changed the references to what applies to Hypha in that file.

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        https://github.com/hyphacoop/<repository-name>/issues
+
+### Submitting Code and Documentation Changes
+
+Our process for accepting changes operates by [Pull Request (PR)](https://help.github.com/articles/about-pull-requests/) and has a few steps:
+
+1.  If you haven't submitted anything before, and you aren't (yet!) a member of our organization, **fork and clone** the repo:
+
+        $ git clone git@github.com:<your-username>/<repository-name>.git
```
Comment (benhylau) [inline contributing.md]:
```suggestion
        $ git clone git@github.com:<your-username>/remote.git
```

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    Organization members should clone the upsteam repo, instead of working from a personal fork:
+
+        $ git clone git@github.com:hyphacoop/<repository-name>.git
```
Comment (benhylau) [inline contributing.md]:
```suggestion
        $ git clone git@github.com:hyphacoop/remote.git
```

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    ![Screenshot of PR button with “draft” option](https://help.github.com/assets/images/help/pull_requests/pullrequest-send.png)
+
+1.  Allow others sufficient **time for review and comments** before merging. We make use of GitHub's review feature to comment in-line on PRs when possible. There may be some fixes or adjustments you'll have to make based on feedback.
+
+    In general, we do our best to provide some feedback or review within about 3 days. (And hopefully much quicker most of the time!)
+    
+    If you have commit rights on the repo, you should merge your own PR, but have someone else review it first:
+    
+    - Allow up to 3 days for review. If you don’t get any feedback by then, you can merge it without review.
```
Comment (benhylau) [inline contributing.md]:
```suggestion
    If you have commit rights on the repo, you should merge your own PR, but have someone else review it first. Allow up to 3 days for review. If you don’t get any feedback by then, you can merge it without review.
```

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Contributing Guidelines
+lang: en
+title-display: EN
```
Comment (benhylau) [inline contributing.md]:
```suggestion
```

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+---
+
```
Comment (benhylau) [inline contributing.md:6]:
```suggestion

_Contributing Guidelines adopted from [Environmental Data and Governance Initiative (EDGI)](https://envirodatagov.org/)._

```

File: contributing.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+### Submitting Feedback, Requests, and Bugs
```
Comment (benhylau) [inline contributing.md:9]:
```suggestion
### Submitting Feedback, Requests, and Bugs

```

File: contributing.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+Our process for submitting feedback, feature requests, and reporting bugs usually begins by opening [GitHub issues](https://help.github.com/articles/about-issues/). Each project repository generally maintains its own set of issues:
+
+        https://github.com/hyphacoop/<repository-name>/issues
```
Comment (benhylau) [inline contributing.md]:
```suggestion
```

File: contributing.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Contributing Guidelines
+lang: en
+title-display: EN
+---
+
+### Submitting Feedback, Requests, and Bugs
+Our process for submitting feedback, feature requests, and reporting bugs usually begins by opening [GitHub issues](https://help.github.com/articles/about-issues/). Each project repository generally maintains its own set of issues:
```
Comment (benhylau) [inline contributing.md]:
```suggestion
This resource accepts feedback, feature requests, and bug reports by opening [GitHub issues](https://github.com/hyphacoop/remote/issues).
```

File: contributing.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
```
Comment (benhylau) [inline contributing.md:11]:
```suggestion
```

**Review by benhylau — CHANGES_REQUESTED** [review]
Please see comments.

**Review by benhylau — APPROVED** [review]

**Pull request merged**